mta/entities/analysis_results.py

```diff
@@ -73,6 +73,8 @@ def is_displayed(self):
 class AnalysisDetailsView(BaseLoggedInPage):
     """This view represents details page of specific analysis"""
 
+    title = Text(".//h1")
```

Comment:
no text present ?

Reply:
Nope. there is no `text` attribute inside `h1` tag.